js/dialogDragAndDrop.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+(function () {
+
+  window.dialog.dialogHandleNode.addEventListener('mousedown', function (evt) {
+    evt.preventDefault();
+
+    var startCoords = {
+      x: evt.clientX,
+      y: evt.clientY
+    };
+
+    var dragged = false;
+
+    var onMouseMove = function (moveEvt) {
+      moveEvt.preventDefault();
+
+      dragged = true;
+
+      var shift = {
+        x: startCoords.x - moveEvt.clientX,
+        y: startCoords.y - moveEvt.clientY
+      };
+
+      startCoords = {
+        x: moveEvt.clientX,
+        y: moveEvt.clientY
+      };
+
+      window.dialog.userDialogNode.style.top = (window.dialog.userDialogNode.offsetTop - shift.y) + 'px';
+      window.dialog.userDialogNode.style.left = (window.dialog.userDialogNode.offsetLeft - shift.x) + 'px';
+
+    };
+
+    var onMouseUp = function (upEvt) {
+      upEvt.preventDefault();
+
+      document.removeEventListener('mousemove', onMouseMove);
+      document.removeEventListener('mouseup', onMouseUp);
+
+      if (dragged) {
+        var onClickPreventDefault = function (clickEvt) {
+          clickEvt.preventDefault();
+          window.dialog.dialogHandleNode.removeEventListener('click', onClickPreventDefault);
+        };
+        window.dialog.dialogHandleNode.addEventListener('click', onClickPreventDefault);
+      }
+    };
```

Comment:
Если обработчик удаляется сразу после первого срабатывания, можно использовать `{once: true}` третьим аргументом. Так он самоликвидируется после выстреливания
```suggestion
      if (dragged) {
        window.dialog.dialogHandleNode.addEventListener('click', function (clickEvt) {
          clickEvt.preventDefault();
        }, {once: true});
    };
```